code/style.css

```diff
@@ -56,3 +108,27 @@ button {
   display: block;
   margin: 0 auto;
 }
+
+button:hover {
+  opacity: 0.7;
+}
+
+button:active {
+  box-shadow: 0 5px #666;
+  transform: translateY(4px);
+}
+
+img {
+  width: 70%;
+  height: auto;
+  margin-top: 30px;
+}
+
+footer{
```

Comment:
Alright, now I understand what the footer tag was for in HTML. 😊 Nice effect with the border around the decks and the white background! When the screen is resized, the footer moves up to the half of the screen, could this be because of the height: 50px property?